cmd/update-operator/main.go

```diff
@@ -25,12 +27,16 @@ var (
 )
 
 func main() {
+	flag.Var(&preRebootActions, "require-pre-reboot", "list of comma-separated kubernetes annotations required to be set to 'true' before the operator will authorize a reboot")
+	flag.Var(&postRebootActions, "require-post-reboot", "list of comma-separated kubernetes annotations required to be set to 'true' before the operator will consider a reboot successful")
```

Comment:
after-reboot-annotations - "List of comma separated Kubernetes node annotations that must be set to true before a node is marked scheduleable and the operator lock is released."